services/provider/server/server.go

```diff
@@ -664,6 +672,14 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
 					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
 				})})
+
+			extR = append(extR, &pb.ExternalResource{
+				Name: "cephfs",
+				Kind: "VolumeGroupSnapshotClass",
+				Data: mustMarshal(map[string]string{
+					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
+					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
```

Comment:
`"csi.storage.k8s.io/snapshotter-secret-name"` to `"csi.storage.k8s.io/group-snapshotter-secret-name"`, @raaizik we need to rethink about sending this information from provider to consumer cluster because this information already exists on the consumer cluster like clusterID it decided on the consumer cluster, not on the provider cluster and same with the secret as well. we need to break a few things like this. cc @nb-ohad

Reply:
@Madhu-1 I agree, although I am not sure how to go about this. Let's look at `ClusterID`: for storage classes of type cephFS, it seems like the server generates it with data pulled from `CephFilesystemSubVolumeGroup`. For that of type cephRBD, it stores in it the namespace of `OCSProviderServer`. Now, on the client both of those get overridden with `storageClaimHash`. Which side is correct?

Reply:
it doesnt make sense to send the information from provider to consumer which is solely decided on the consumer cluster not on the provider cluster. https://github.com/red-hat-storage/ocs-client-operator/blob/3165a1b7db9b73661099f89005ed271690234240/controllers/storageclaim_controller.go#L373-L379 this is complete decided on the consumer, whatever the information sent from provider its not used anywhere in the provider cluster.

Reply:
That is obvious, my question was which side is correct, not what is the eventual value that's picked up. Asking because it looks too [elaborate](https://github.com/red-hat-storage/ocs-operator/blob/main/services/provider/server/server.go#L428-L437) to be coincidental.

Reply:
not sending it on the provider but setting it up at the consumer seems correct to me.

Reply:
@nb-ohad 

> on the client the clusterID of both of those gets overridden with storageClaimHash

Reply:
Final decision is to remove `clusterID` calculation here. @Madhu-1 regarding `csi.storage.k8s.io/snapshotter-secret-name`: it isn't assigned in the consumer, only `csi.storage.k8s.io/snapshotter-secret-namespace` is.